media/common/src/main/java/io/helidon/media/common/MessageBodyWriterContext.java

```diff
@@ -118,7 +117,7 @@ private MessageBodyWriterContext(MessageBodyWriterContext writerContext, Paramet
         this.headers = headers;
         this.writers = new MessageBodyOperators<>(writerContext.writers);
         this.swriters = new MessageBodyOperators<>(writerContext.swriters);
-        this.acceptedTypes = writerContext.acceptedTypes;
+        this.acceptedTypes = List.copyOf(writerContext.acceptedTypes);
```

Comment:
As far as I can see, `List.copyOf` actually gets all the elements in the list. So the lazyness would be gone in this case.

Reply:
I am not sure if we do need the copy, it's  unmodifiable list all the time, but I could have missed something WDYT @romain-grecourt ?

Reply:
Right, this list shouldn't need to be copied.

Reply:
fixed